holo-config-js/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "holo_config_js"
+version = "0.0.0"
+license = "Apache-2.0"
+
+[lib]
+name = "holo_config_js"
+crate-type = ["cdylib"]
+
+[dependencies]
+holo-config = { path = ".." }
+failure = "0.1.5"
+serde_json = "1.0.39"
+wasm-bindgen = "=0.2.40"
+wee_alloc = "=0.4.4"
```

Comment:
Strict dependency matches cause issues with downstream depending on that crate. We commit `Cargo.lock`, which guarantees reproducibility.